testcafe/tests/drive/classification_scenario.js

```diff
@@ -213,7 +212,7 @@ test(`${TEST_MOVE_FILE}`, async t => {
     `${FEATURE_PREFIX}/${TEST_MOVE_FILE}-1`,
     true
   )
-  await moveMoadal.moveElementTo(`${FEATURE_PREFIX}-Folder1`)
+  await privateDrivePage.moveElementTo(`${FEATURE_PREFIX}-Folder1`)
```

Comment:
Shouldn't this use the second parameter of `moveElementTo`, `inModal`?

Reply:
`moveElementTo` doesn't have a second parameter. It call method like `goToFolder` which have a `isModal` parameter.

```
async moveElementTo(destinationFolder) {
    await this.goToBaseFolder(true)
    await this.goToFolder(destinationFolder, true)

    await isExistingAndVisibile(selectors.btnModalSecondButton, 'Move Button')

    await t.click(selectors.btnModalSecondButton)
  }
```